<launch>
  <include file="$(find robo_magellan)/launch/cone_finder.launch"/>
  <node name="cone_seeker" pkg="robo_magellan" type="seek_cone.py"/>
</launch>

